jobs/github-event/installation/created.js

```diff
@@ -10,7 +10,12 @@ const upsert = require('../../../lib/upsert')
 
 module.exports = async function ({ installation }) {
   const { installations, repositories: reposDb, logs } = await dbs()
-  const log = Log({logsDb: logs, accountId: installation.account.id, repoSlug: null, context: 'integration-installation-created'})
+  const log = Log({
```

Comment:
🎨  for next time: separate functional changes and whitespace changes into different commits.

Reply:
that way it is easier to find that the change was the `context` string